packages/deployer/test/unit/internal/process-transactions.test.js

```diff
@@ -69,9 +69,9 @@ describe('internal/process-transactions.js', function () {
       });
 
       it('registers a transaction in the deployment data', async function () {
-        const transaction = hre.deployer.deployment.general.transactions[transaction.hash];
-        equal(transaction.status, 'confirmed');
-        equal(transaction.description, 'test');
+        const tx = hre.deployer.deployment.general.transactions[transaction.hash];
```

Comment:
This was also fixed in another PR. Not sure how this made it to main.

Reply:
Probably a back merge of the `main` branch?

Reply:
The other fix is not merged yet (PR 513)